proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -539,7 +540,12 @@ message ActivityInfo {
 
     // The deployment this activity was dispatched to most recently. Present only if the activity
     // was dispatched to a versioned worker.
+    // Deprecated. Replaced by last_deployment_version.
     temporal.api.deployment.v1.Deployment last_started_deployment = 43;
+
+    // The deployment this activity was dispatched to most recently. Present only if the activity
+    // was dispatched to a versioned worker.
+    string last_deployment_version = 44;
```

Comment:
Nit: should change to `last_worker_deployment_version` to be consistent with the public protos.